cli/src/main/java/io/quarkiverse/backstage/cli/common/GenerationBaseCommand.java

```diff
@@ -32,7 +32,8 @@ public abstract class GenerationBaseCommand<T> extends BackstageClientAwareComma
     private static final ArtifactDependency QUARKUS_BACKSTAGE = new ArtifactDependency("io.quarkiverse.backstage",
             "quarkus-backstage", null, "jar", GenerationBaseCommand.getVersion());
 
-    @Option(names = { "--namespace" }, description = "The target namespace (where the Custom Resources will be installed)")
+    @Option(names = {
+            "--namespace" }, description = "The target namespace (where the Custom Resources will be installed). Defaults to 'default'")
```

Comment:
Should we use such a wording "where the Custom Resources will be installed". AFAIK, all the backstage resources are YAML files but they are not mapped to any kubernetes custom resources and processed as such on the platform